generative/maisi/data/README.md

```diff
@@ -0,0 +1,107 @@
+
+In total, we included:
+| Index | Dataset Name                                   | Number of Training Data | Number of Validation Data |
+|-------|------------------------------------------------|-------------------------|---------------------------|
+| 1     | Covid 19 Chest CT                              | 722                     | 49                        |
+| 2     | TCIA Colon Abdomen CT                          | 1522                    | 77                        |
+| 3     | MSD03 Liver Abdomen CT                         | 104                     | 0                         |
+| 4     | LIDC chest CT                                  | 450                     | 24                        |
+| 5     | TCIA Stony Brook Covid Chest CT                | 2644                    | 139                       |
+| 6     | NLST Chest CT                                  | 31801                   | 1674                      |
+| 7     | TCIA Upenn GBM Brain MR (skull-stripped)       | 2550                    | 134                       |
+| 8     | Aomic Brain MR                                 | 2630                    | 138                       |
+| 9     | QTIM Brain MR                                  | 1275                    | 67                        |
+| 10    | Acrin Chest MR                                 | 6599                    | 347                       |
+| 11    | TCIA Prostate MR Below-Abdomen MR              | 928                     | 49                        |
+| 12    | Aomic Brain MR, skull-stripped                 | 2630                    | 138                       |
+| 13    | QTIM Brain MR, skull-stripped                  | 1275                    | 67                        |
+|       | Total CT                                       | 37243                   | 1963                      |
+|       | Total MRI                                      | 17887                   | 940                       |
+
+
+### 2 Diffusion model training Data
```

Comment:
Hi @dongyang0122, I guess you may also help update this part.

Reply:
I have updated this part. Please proceed. Thanks.

Reply:
Could you please also mention the number for the training and validation? Thanks!

Reply:
If I remember correctly, all data are used for training and there is no validation data? Please correct me if I'm wrong. @dongyang0122

Reply:
@Can-Zhao, all data points are used for model training.